docs/site/DEVELOPING.md

```diff
@@ -549,6 +549,40 @@ repository.
 
 ### Create a new package
 
+Please run the following command:
+
+```sh
+cd loopback-next
+node bin/create-package.js
+```
+
+The script does the following steps:
+
+1.  Determine the location and package name.
+
+    The first argument of the command can be one of the following:
+
+    - package-name
+    - @loopback/package-name
+    - extensions/package-name
+    - packages/package-name
+
+    If the location is not specified, it tries to guess by the current directory
+    and falls back to `extensions`.
+
+2.  Run `lb4 extension` to scaffold the project without `npm install`. If
+    `--yes` or `-y` is provide by the command, interactive prompts are skipped.
+
+3.  Tidy up the project
+
+    - Remove unused files
+    - Rename `tsconfig.json` to `tsconfig.build.json`
```

Comment:
@raymondfeng I think this is no longer accurate, now that we are using project references (via https://github.com/strongloop/loopback-next/pull/5155)?